README.md

```diff
@@ -152,6 +152,9 @@ Tests if the value is valid json.
 ### NotEmpty
 Tests if the value is not an empty (empty object, empty array, empty string, etc).
 
+### NullOrDate
+Tests if the value is a `null` or `date`.
```

Comment:
If so we need to update [here](https://github.com/seegno/validator.js-asserts/pull/102/files#diff-04c6e90faac2675aa89e2176d2eec7d8R94) as well.